specification/synapse/data-plane/Microsoft.Synapse/stable/2023-01-01/linkConnections.json

```diff
@@ -0,0 +1,1011 @@
+      "description": "The request of edit link table(s)",
+      "properties": {
+        "linkTables": {
+          "type": "array",
+          "description": "Edit link tables request",
+          "items": {
+            "$ref": "#/definitions/LinkTableRequest"
+          }
+        }
+      }
+    },
+    "LinkTableRequest": {
+      "type": "object",
+      "description": "The request of a link table",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Link table id"
+        },
+        "source": {
+          "$ref": "#/definitions/LinkTableRequestSource",
+          "description": "Source table properties for link table request"
+        },
+        "target": {
+          "$ref": "#/definitions/LinkTableRequestTarget",
+          "description": "Target table properties for link table request"
+        },
+        "operationType": {
+          "type": "string",
+          "description": "Link table operation type"
+        }
+      }
+    },
+    "LinkTableRequestSource": {
+      "type": "object",
+      "description": "The source request of a link table",
+      "properties": {
+        "tableName": {
+          "type": "string",
+          "description": "Source table table name"
+        },
+        "schemaName": {
+          "type": "string",
+          "description": "Source table schema name"
+        }
+      }
+    },
+    "LinkTableRequestTarget": {
+      "type": "object",
+      "description": "The target request of a link table",
+      "properties": {
+        "tableName": {
+          "type": "string",
+          "description": "Target table table name"
+        },
+        "schemaName": {
+          "type": "string",
+          "description": "Target table schema name"
+        },
+        "distributionOptions": {
+          "$ref": "#/definitions/LinkTableRequestTargetDistributionOptions",
+          "description": "Target table distribution options for link table request"
+        },
+        "structureOptions": {
+          "$ref": "#/definitions/LinkTableRequestTargetStructureOptions",
+          "description": "Target table structure options for link table request"
+        }
+      }
+    },
+    "LinkTableRequestTargetDistributionOptions": {
+      "type": "object",
+      "description": "The distribution options of a target table",
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "Target table distribution type"
+        },
+        "distributionColumn": {
+          "type": "string",
+          "description": "Target table distribution column"
+        }
+      }
+    },
+    "LinkTableRequestTargetStructureOptions": {
+      "type": "object",
+      "description": "The structure options of a target table",
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "Target table structure type"
+        }
+      }
+    },
+    "LinkConnectionDetailedStatus": {
+      "type": "object",
```

Comment:
Are any of the properties supposed to be required?

Reply:
LinkConnectionDetailedStatus is required by "/linkconnections/{linkConnectionName}/detailedstatus"